### DISCUSSION TITLE
please i am really i need of help badly

### DISCUSSION THREAD

**Discussion by King7474** [WalletConnect/walletconnect-monorepo#2989] [Developer Support]
i have been trying to upgrade to v2 for days but not working 
errors install npm i 
errors on package json 
i am dead off 
i dont know what to do anymore

**Comment by crypblizz8** [comment]
Sorry to hear about this experience. Can you share more details about your setup and your error so we can can help you?

**Reply by King7474** [reply]
error on typescript 
i need a manual set on dependences needed to get a good working walletconnect v2 
and can authenticate any wallet and devices successfully without errors or problems 
and also, import files and codes to use on my server or rathe a js code i can import on my index html to make it work 

i have done a lot of npm install from instructions but i am frustrated rn 
i need help me please

**Reply by King7474** [reply]
i was using web3 provider 
the wallet connect options for wallets are coming up 
but non is connecting to any wallet successfully

**Comment by King7474** [comment]
i wish i can send the file here 
but it is a v1 file
